src/Symfony/Component/Image/README.md

```diff
@@ -0,0 +1,3 @@
+Symfony Image Component
+=======================
+
```

Comment:
The readme is incomplete here. It should have the basic info we have in other components